src/site/_includes/partials/analytics.njk

```diff
@@ -1,10 +1,13 @@
 {% set analyticsScript %}
 window.dataLayer = window.dataLayer || [];
-function gtag(){dataLayer.push(arguments);}
-gtag('js', new Date());
+window.dataLayer.push({'gtm.start': new Date().getTime(),event:'gtm.js'});
+window.dataLayer.push({'user_agent': navigator.userAgent});
+//Restrict GTM variables:
+dataLayer.push({'gtm.blacklist': ['html', 'd', 'jsm', 'j']});
+function gtag(){dataLayer.push(arguments)};
 {% if (site.env != 'prod') %}
   {# In development, log debug events to the console instead of sending them to GA #}
-  window.gtag = console.debug;
+  //window.gtag = console.debug;
```

Comment:
Uncomment this before merging.

Or alternatively remove this, now we have staging setup.